<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room</class>
 <widget class="QDialog" name="room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/source/picture/icon/room_icon.jpg</normaloff>:/source/picture/icon/room_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/source/picture/background/room.jpg);</string>
  </property>
  <widget class="QWidget" name="card_1" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>80</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
  </widget>
  <widget class="QWidget" name="card_2" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>80</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
  </widget>
  <widget class="QWidget" name="card_3" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>80</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
  </widget>
  <widget class="QWidget" name="card_4" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>80</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
  </widget>
  <widget class="QWidget" name="card_5" native="true">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>80</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
  </widget>
  <widget class="QLabel" name="timelabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
